net-p2p/xmr-stak/files/xmr-stak

```diff
@@ -0,0 +1,24 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms fo the GNU General Public License, v2
+# /etc/init.d/xmr-stak
+name="xmr-stak"
+description="Unified all-in-one Monero miner"
+pidfile=/var/run/xmr.pid
```

Comment:
For modern applications I prefer just `/run` but ofc, you keep using `/var/run`.